scenes/game/local_game.gd

```diff
@@ -4975,6 +4998,8 @@ func handle_strike_effect(card_id :int, effect, performing_player : Player):
 			performing_player.life = amount
 			events += [create_event(Enums.EventType.EventType_Strike_GainLife, performing_player.my_id, amount, "", performing_player.life)]
 			_append_log_full(Enums.LogType.LogType_Health, performing_player, "gains %s life, bringing them to %s!" % [str(amount), str(performing_player.life)])
+		"shuffle_discard_in_place":
+			performing_player.random_shuffle_discard_in_place()
```

Comment:
hmm, do we need an event for this?
Otherwise the top discard card won't be correct i think?
Probably need some event to reorder the discards.

Reply:
sorry, what do you mean by top discard card being incorrect?